content/concepts/multiplex/yamux.md

```diff
@@ -1,15 +1,22 @@
 ---
-title: "yamux"
+title: "Yamux"
 description: "yamux is a multiplexing protocol designed by Hashicorp."
 weight: 180
 ---
 
-[yamux](https://github.com/hashicorp/yamux) is a multiplexing protocol designed by [Hashicorp](https://www.hashicorp.com/).
-yamux offers more sophisticated flow control than mplex, and can scale to thousands of multiplexed streams over a
-single connection.
+## What is Yamux?
 
-{{< alert icon="💡" context="note">}}
-This section is incomplete, and many of the articles are stubs. To help fill in
-the gaps, please see the issues linked in each article to add your input and
-help us prioritize the outstanding work.
-{{< /alert >}}
+[Yamux](https://github.com/hashicorp/yamux) is a multiplexing protocol designed
+by [Hashicorp](https://www.hashicorp.com/) for Go. It is built over
+[SCTP (Stream Control Transmission Protocol)](https://en.wikipedia.org/wiki/Stream_Control_Transmission_Protocol), which is a transport layer protocol
+that provides reliable, message-oriented communication with congestion control.
+Yamux provides a high-level interface for working with SCTP streams. It is inspired
+by [SPDY](https://en.wikipedia.org/wiki/SPDY), but is not interoperable with it.
+
+Yamux offers more sophisticated flow control than [mplex](mplex), and can scale
+to thousands of multiplexed streams over a single connection.
```

Comment:
mplex can also handle 1000s of streams. We should say that it does support backpressure, and maybe explain how backpressure works (the receiver allows sending of stream data up to a certain offset, and that offset is increased as data is processed by the application), and why you need backpressure (application might be slow processing the data, and we don't want to buffer an unlimited amount of data).